src/overlays/actors/ovl_En_Nb/z_en_nb.h

```diff
@@ -6,11 +6,84 @@
 
 struct EnNb;
 
+typedef void (*EnNbActionFunc)(struct EnNb*, GlobalContext*);
+typedef void (*EnNbDrawFunc)(struct EnNb*, GlobalContext*);
+
 typedef struct EnNb {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0x1DC];
+    /* 0x014C */ SkelAnime skelAnime;
+    /* 0x0190 */ Vec3s limbDrawTable[19];
+    /* 0x0202 */ Vec3s transitionDrawTable[19];
+    /* 0x0274 */ s16 eyeIdx;
+    /* 0x0276 */ s16 blinkTimer;
+    /* 0x0278 */ s32 action;
+    /* 0x027C */ s32 drawMode;
+    /* 0x0280 */ f32 alphaTimer;
+    /* 0x0284 */ u32 alpha;
+    /* 0x0288 */ s32 flag;
+    /* 0x028C */ s32 unk_28C; // some sort of action
```

Comment:
I suggest `previousCsAction` for `unk_28C`, it's used to prevent playing a cutscene action multiple times in a row (since each cutscene action may not be reset)